test/api_stability/3.0.2/0002-NOAPI-DETAIL-MISC-Remove-top-level-scoring-scheme-de.patch

```diff
@@ -20,7 +20,7 @@ index ef141a7e9..7b0927215 100644
 -                                                                seqan3::align_cfg::method_global,
 -                                                                seqan3::aminoacid_scoring_scheme<>>;
 +                                                                seqan3::align_cfg::method_global>;
- 
+
```

Comment:
This doesn't look necessary for a BLOSUM80 -> blosum80 change.
Did this accidentally get into the commit? Or is it something that was forgotten before?